packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -409,7 +409,7 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
   private _handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>): void => {
     // eat the up and down arrow keys to keep focus in the spinButton
     // (especially when a spinButton is inside of a FocusZone)
-    if (event.which === KeyCodes.up || event.which === KeyCodes.down) {
+    if (event.which === KeyCodes.up || event.which === KeyCodes.down || event.which === KeyCodes.enter) {
```

Comment:
and ENTER still submits the change in the spinButton?

Reply:
@chang47 can you confirm this? I don't want to merge until we're sure this functionality is still the same.

Reply:
@micahgodbolt Yes it still submits, however I have one concern. I looked at the spin button behavior on the fabric website: https://developer.microsoft.com/en-us/fabric#/components/spinbutton

and I noticed that when I press enter, the SpinButton text gets selected. I thought I caused a regression, but when I removed the code I added, I noticed that the text still doesn't get selected after pressing enter.

I see that the only way for the text to get selected would be in the onFocus() function, so I believe there seems to be some sort of discrepancy.

What is the expected behavior for this?

Reply:
I'm seeing the text get selected when I press ENTER on the fabric website, which is expected.

Did you make sure you didn't have cached js when you tested with your change removed?

The expected behavior is that the text is selected when ENTER is pressed

Reply:
I tried reproing this behavior directly from the main fabric repo and the text is still not selected after pressing enter, I believe someone must have introduced a regression from a different PR.

I believe the reason is that we're not calling the onFocus code after we press enter. I don't think me stopping a keyboard event from propagating would stop a focus event. So I think what I will do is merge my change in and create an issue. 

Is that acceptable to you?